addons/l10n_in_pos/static/src/overrides/models/closing_session.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module */
+
+import { ClosePosPopup } from "@point_of_sale/app/navbar/closing_popup/closing_popup";
+
+import { _t } from "@web/core/l10n/translation";
+import { patch } from "@web/core/utils/patch";
+import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+
+patch(ClosePosPopup.prototype, {
+    async confirm() {
+        if (this.pos.company.country_id?.code === "IN" && !this.pos.company.state_id) {
+            this.dialog.add(ConfirmationDialog, {
+                title: _t("Company address is missing"),
+                body: _t(
+                    "Your company %s needs to have a correct address in order validate this invoice.\n" +
```

Comment:
Validate this invoice ? Weird message when closing the session

Reply:
After discussion with PO. This message is uses in multiple times.